docs/troubleshooting/README.md

```diff
@@ -46,32 +46,35 @@ More information on kinesis error handling is [here](data-cookbooks/cnm-workflow
 
 `KMS Exception: AccessDeniedExceptionKMS Message: The ciphertext refers to a customer master key that does not exist, does not exist in this region, or you are not allowed to access.`
 
-The above error was being thrown by cumulus lambda function invocation. The KMS key is the encryption key used to encrypt lambda environment variables. The root cause of this error is unknown.
+The above error was being thrown by cumulus lambda function invocation. The KMS key is the encryption key used to encrypt lambda environment variables. The root cause of this error is unknown, but is speculated to caused by deleting and recreating, with the same name, the IAM role the lambda uses.
 
 On a lambda level, this error can be resolved by updating the KMS Key to `aws/lambda`. We've done this through the management console. Unfortunately, this approach doesn't scale well.
```

Comment:
FWIW, I investigated trying to use this solution yesterday and it doesn't seem to work or make any sense? I would vote for removing it

Reply:
Let me update it to what worked for Abdelhak yesterday.

Reply:
eb775a478